sst_wrapper/utils/robot_state_factory.py

```
@@ -54,3 +66,5 @@ def get_robot_state_info(robot_name, world_size, origin):
             ],
             [False, False, True, False, False],
         )
+
+    return []
```

probably better to raise ValueError("unknown robot type")